src/main/java/uk/gov/register/resources/HomePageResource.java

```diff
@@ -51,4 +58,15 @@ public String analyticsTrackingId() {
                 trackingId -> "var gaTrackingId = \"" + trackingId + "\";\n"
         ).orElse("");
     }
+
+    @DELETE
+    @PermitAll
+    @Path("/delete-register-data")
+    @DataDeleteNotAllowed
+    public Response deleteRegisterData() {
```

Comment:
I wonder if this makes sense to be a separate class, rather than shoving it into the HomePageResource?